web/app.js

```diff
@@ -560,6 +560,13 @@ const PDFViewerApplication = {
 
     if (appConfig.annotationEditorParams) {
       if (annotationEditorMode !== AnnotationEditorType.DISABLE) {
+        if (AppOptions.get("enableStampEditor") === true) {
+          const editorStampButton = document.getElementById("editorStamp");
+          if (editorStampButton) {
+            editorStampButton.hidden = false;
+          }
+        }
+
```

Comment:
We should already have a reference to the DOM element, so please use that here :-)
Furthermore, the Stamp-editor uses `OffScreenCanvas` unconditionally so we *also* need to check that that's supported before displaying the button.
```suggestion
        const editorStampButton = appConfig.toolbar?.editorStampButton;
        if (
          editorStampButton &&
          AppOptions.get("enableStampEditor") &&
          AppOptions.get("isOffscreenCanvasSupported") &&
          FeatureTest.isOffscreenCanvasSupported
        ) {
          editorStampButton.hidden = false;
        }
```